oauth_dropins/appengine_config.py

```diff
@@ -90,6 +91,7 @@ def read(filename):
 FLICKR_APP_SECRET = read('flickr_app_secret')
 GOOGLE_CLIENT_ID = read('google_client_id')
 GOOGLE_CLIENT_SECRET = read('google_client_secret')
+INDIEAUTH_CLIENT_ID = read('indieauth_client_id')
```

Comment:
true, fair question. i guess we'd override `StartHandler.to()` and add a kwarg, like twitter's does.

i don't feel strongly though. if it doesn't feel right, i'm ok with using a file.